transcom/mymove
# [MB-15792] adding created and updated at columns to sit_address_updates table

## [Jira ticket](tbd)

## Summary

Migration that adds `created_at` and `updated_at` columns to `sit_address_updates` table and removes the `reason` column along with associated updates to models, factories, and tests


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Run `make db_dev_migrate`
2. new columns should appear and the `reason` column should no longer exist
3. Run `make server_test` and all tests should pass

### Backend


### Database

#### Any new migrations/schema changes:
